README.rst

```diff
@@ -70,7 +70,7 @@ Documentation
 =============
 PyOTA's documentation is available on `ReadTheDocs`_.
 
-If you are :ref:`installing from source <readme-installing-from-source>`, you
+If you are [installing from source](#installing-from-source), you
```

Comment:
The previous syntax is actually correct for `.rst` files.  I'll file a request with GitHub support to figure out why it's not rendering correctly.

Reply:
Hey Marius.  I heard back from GitHub support.

Turns out that `:ref:` is not actually part of the reStructuredText spec; it's a custom feature provided by Sphinx.

I made the following changes:
- Removed the link.
- Added a metadata check to `.travis.yml` (which caught the `:ref:` right awy!).

Thanks for drawing attention to this issue!